TMS.Net07.Homework4.RecursionAndFibonacci/TMS.Net07.Homework4.Task4.Fibonacci/TMS.Net07.Homework4.Task4.Fibonacci/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace TMS.Net07.Homework4.Task4.Fibonacci
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            while (true)
+            {
+                Console.Write("Input a number to calculate a factorial: ");
+                long inputNumber = 0;
+                bool isTryInputNumber = false;
+
+                do
+                {
+                    isTryInputNumber = long.TryParse(Console.ReadLine(), out inputNumber);
+
+                    if (isTryInputNumber == false)
```

Comment:
```suggestion
                    if (!isTryInputNumber)
```
Here and other same places

Reply:
I fixed all the mistakes